assets/js/components/thesis-editor.js

```diff
@@ -468,6 +478,7 @@ class ThesisEditor extends React.Component {
         </div>
         <div id='thesis-fader' className={this.renderFaderClass()} />
         <div id='thesis-tray' className={this.renderTrayClass()}>
+          <TrayNotifications notifications={this.parseNotifications(this.state.trayType)} />
```

Comment:
We pass in notifications the TrayNotifications component. Since we have a parseNotifications() functions, we can have more than just tray notifications down the road.